#src/scss/blocks/form.scss

```
@@ -0,0 +1,32 @@
+.form__title {
+    font-family: $fontSecondary;
+    font-size: 22px;
+    font-weight: 600;
+    color: $colorDark;
+    margin-bottom: 48px;
+    line-height: 1.3em;
+    position: relative;
+}
+.form__title::before {
+    content: "";
+    background: url("../../img/circle.svg") no-repeat;
+    width: 26px;
+    height: 26px;
+    position: absolute;
+    right: 529px;
```

same as hero__header, refer to comment for `hero__header::after`
use smth like
```suggestion
    left: -10px; // or whatever negative number instead of -10
```